### PR TITLE
update map styling and add dummy oecm and wdpa layers

### PR DESCRIPTION
**Pull request by WKocur**
Updates map styling and removes the download button as agreed.

Note that the wdpa and oecm layers are currently dummy, but this can still be reviewed.

### CODE REVIEW

**Comment by WKocur** [comment]
Can be merged - but make sure if on staging everyone is aware that the PAs and OECM areas are not yet pulling through on the map.

File: app/assets/stylesheets/styles/components/map/_map.scss [hunk 1]
```diff
@@ -1,42 +1,56 @@
...
 
+$map-filters-button-bar-height: rem-calc(92);
+
 .map {
   &__container {
     margin-bottom: rem-calc(35);
-    height: 600px;
-
-    flex-direction: row-reverse;
+    border: $border-thin $navy;
```
Comment (nym1k) [inline app/assets/stylesheets/styles/components/map/_map.scss:9]: Property order

File: app/assets/stylesheets/styles/components/map/_map.scss [hunk 2]
```diff
@@ -118,6 +137,7 @@
         margin-right: $key-margin;
         width: $key-size; height: $key-size;
 
+        position: relative;
         display: block;
```
Comment (nym1k) [inline app/assets/stylesheets/styles/components/map/_map.scss:141]: Property order? I think position goes after display

File: app/assets/stylesheets/styles/components/map/_map.scss [hunk 3]
```diff
@@ -237,28 +269,59 @@
 
 .eez-map-legend {
   @include flex;
-  @include flex-v-center;
-  @include flex-h-around;
+  @include flex-column;
+  padding: rem-calc(15 17);
   font-size: 0.7rem;
```
Comment (nym1k) [inline app/assets/stylesheets/styles/components/map/_map.scss:274]: Prop order

File: app/assets/stylesheets/styles/components/map/_map.scss [hunk 4]
```diff
@@ -237,28 +269,59 @@
...
-  height: 11%;
   width: 100%;
 
+  @include breakpoint($small) {
+    padding-left: 0;
+    padding-right: 0;
```
Comment (nym1k) [inline app/assets/stylesheets/styles/components/map/_map.scss:279]: Property order

File: app/views/site/partials/_eez_map.html.erb [hunk 5]
```diff
@@ -11,13 +11,13 @@
     <% end %>
     <v-map-interactive
       :id="'eez-map'"
-      :has-download-button="true"
+      :nload-button="true"
```
Comment (nym1k) [inline app/views/site/partials/_eez_map.html.erb:14]: nload? Is that correct?

**Review by nym1k — CHANGES_REQUESTED** [review]
Sorry, just being fussy with CSS property orders 😅

File: app/views/site/partials/_eez_map.html.erb (hunk 5 above)
Comment (WKocur) [reply app/views/site/partials/_eez_map.html.erb:14]: A failed attempt to delete the whole line! 🙃

**Pull request merged**